### PR TITLE
Chat attachment view lightbox popup

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 9c61437e1414...3088c46c5a4a introduce visual changes on linux. <details><summary>:mag_right: 5 new</summary>

 * Added: **Popup_Dialog-Normal**  
   Popup_Dialog-Normal rendered
 * Added: **Attachment_Popup-You_Zoomed**  
   Attachment_Popup-You_Zoomed rendered
 * Added: **Attachment_Popup-You_Popup_Showing**  
   Attachment_Popup-You_Popup_Showing rendered
 * Added: **Attachment_Popup-You**  
   Attachment_Popup-You rendered
 * Added: **Attachment_Popup-Following_Popup_Showing**  
   Attachment_Popup-Following_Popup_Showing rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 9c61437e1414...3088c46c5a4a introduce visual changes on linux. <details><summary>:mag_right: 5 new</summary>

 * Added: **Popup_Dialog-Normal**  
   Popup_Dialog-Normal rendered
 * Added: **Attachment_Popup-You_Zoomed**  
   Attachment_Popup-You_Zoomed rendered
 * Added: **Attachment_Popup-You_Popup_Showing**  
   Attachment_Popup-You_Popup_Showing rendered
 * Added: **Attachment_Popup-You**  
   Attachment_Popup-You rendered
 * Added: **Attachment_Popup-Following_Popup_Showing**  
   Attachment_Popup-Following_Popup_Showing rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 9c61437e1414...f4c5c7b7970d introduce visual changes on linux. <details><summary>:mag_right: 7 new</summary>

 * Added: **Popup_Dialog-Normal**  
   Popup_Dialog-Normal rendered
 * Added: **Attachment_Popup-You_Zoomed**  
   Attachment_Popup-You_Zoomed rendered
 * Added: **Attachment_Popup-You_Wide_Image**  
   Attachment_Popup-You_Wide_Image rendered
 * Added: **Attachment_Popup-You_Small_Image**  
   Attachment_Popup-You_Small_Image rendered
 * Added: **Attachment_Popup-You_Popup_Showing**  
   Attachment_Popup-You_Popup_Showing rendered
 * Added: **Attachment_Popup-You**  
   Attachment_Popup-You rendered
 * Added: **Attachment_Popup-Following_Popup_Showing**  
   Attachment_Popup-Following_Popup_Showing rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 9c61437e1414...4e97beadba44 introduce visual changes on linux. <details><summary>:mag_right: 7 new</summary>

 * Added: **Popup_Dialog-Normal**  
   Popup_Dialog-Normal rendered
 * Added: **Attachment_Popup-You_Zoomed**  
   Attachment_Popup-You_Zoomed rendered
 * Added: **Attachment_Popup-You_Wide_Image**  
   Attachment_Popup-You_Wide_Image rendered
 * Added: **Attachment_Popup-You_Small_Image**  
   Attachment_Popup-You_Small_Image rendered
 * Added: **Attachment_Popup-You_Popup_Showing**  
   Attachment_Popup-You_Popup_Showing rendered
 * Added: **Attachment_Popup-You**  
   Attachment_Popup-You rendered
 * Added: **Attachment_Popup-Following_Popup_Showing**  
   Attachment_Popup-Following_Popup_Showing rendered
</summary>

**Comment by chromakode** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/attachment-popup/container.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 import {navigateUp} from '../../../actions/route-tree'
 
 import type {TypedState} from '../../../constants/reducer'
 
-export default connect(
-  (state: TypedState, {routeProps}) => routeProps,
-  (dispatch: Dispatch) => ({
-    onClose: () => dispatch(navigateUp()),
-  })
+export default compose(
```
Comment (chrisnojima) [inline shared/chat/conversation/attachment-popup/container.js]: You don't have to make this change but when I was doing my recompose work I ended up leaving the connect wrapping in the container.js and did my recompose wrapping in a wrapped export of the renderered component. It cleaned up the connect file (often complex) and had the benefit of making the dumb component function (like the state would actually work in the dumb sheet)

**Pull request merged**